### ISSUE TITLE
How to create SecretId and RoleId objects

### ISSUE THREAD

**Issue by dheerajjoshim** [spring-projects/spring-vault#186]
I want to use wrapped secret ID for App role authentication. I was using 1.1.1 But figured out latest 2.0 version has this support.

Added following dependency and Java code

```
                <dependency>
    		<groupId>org.springframework.vault</groupId>
    		<artifactId>spring-vault-core</artifactId>
    		<version>2.0.0.RC1</version>
  		</dependency>

```
  
```
         AppRoleAuthenticationOptions options = AppRoleAuthenticationOptions.builder()
                .appRole("my_app_role").roleId(RoleId.provided("role_Id"))
                .secretId(SecretId.wrapped(VaultToken.of("wrapped_token"))).build();
        return new AppRoleAuthentication(options, restOperations());

```
In above Java code I am unable to resolve SecretId and RoleId

But how to create SecretId and RoleId objects in @Configuration class? Are there any factory methods provided to create these objects?

**Comment by mp911de** [comment]
That's a known bug in 2.0.0.RC1 which is fixed with #179. Care to give 2.0.0.BUILD-SNAPSHOT a spin?

**Comment by dheerajjoshim** [comment]
It appears to be working. Thanks for confirming
